cypress/e2e/awx/resources/inventories.cy.ts

```diff
@@ -165,58 +166,24 @@ describe('Inventories Tests', () => {
           //Add an assertion that the inventory does not appear upon a list search
         });
 
-        it.skip('can bulk delete inventories from the list view and verify deletion', () => {
+        it('can bulk delete inventories from the list view and verify deletion', () => {
```

Comment:
This is covered in my PR: https://github.com/ansible/ansible-ui/pull/2492.

No need to do this.